api.go

```diff
@@ -210,6 +210,7 @@ type (
 	TaskStatus struct {
 		Phase        string  `json:"phase"`
 		Progress     float64 `json:"progress"`
+		Retries      int     `json:"retries"`
```

Comment:
to be consistent with api
```suggestion
		Retries      int     `json:"retries,omitempty"`
```

Reply:
Right! Thanks